src/endpoints/connectors.ts

```diff
@@ -1,8 +1,9 @@
 import { client } from 'services/client';
+import { callSupabase, supabase } from 'services/supabase';
 import { BaseResponse } from 'types';
 
 export enum ConnectorTypes {
```

Comment:
Out of curiosity, if _capture_ and _materialization_ are connector types, what are _Hello World_ and  _Postgres_ classified as?

Reply:
That's a good question... I am not totally sure. Like I normally think of them as "endpoints". We should ask this to the larger group.